darts/tests/models/forecasting/test_historical_forecasts.py

```diff
@@ -990,10 +995,15 @@ def test_torch_auto_start_multiple_no_cov(self, model_config):
         ), f"Model {model_cls} did not return a list of historical forecasts"
         theorical_forecast_length = (
             self.ts_val_length
-            - bounds[0]  # prediction input sample length
+            - (bounds[0] - 1)  # prediction input sample length
```

Comment:
maybe short comment to explain that the `+1` is because the last valid prediction start point is not the last timestamp in the series but the point after (if the covariates are long enough)